CHANGES.rst

```diff
@@ -4,7 +4,8 @@ Changelog
 5.8.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Use custom creator criteria.
```

Comment:
why you need to use a custom creator criteria? It's better to improve infos here because you will forget it in 5 minutes